services/bettersocial/fcmToken/sendMultiDeviceCommentNotification.js

```diff
@@ -24,10 +24,19 @@ const sendMultiDeviceCommentNotification = async (
         data: {
           feed_id: activity_id,
           type: 'feed'
-        }
+        },
+        token: user?.token
       };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code assumes that `user` object will always have a `token` property. If the `user` object does not have a `token` property, it could lead to unexpected behavior or errors. It would be better to check if the `token` property exists before using it.
```diff
-        token: user?.token
+        token: user?.token || ''
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->